src/components/header/header.hbs

```diff
@@ -2,7 +2,7 @@
 <header class="global-header" data-component="global-header">
   <a class="global-header__logo" href="{{rootPath}}/index.html">
     <span class="global-header__site-icon" aria-hidden="true">A</span>
-    <h1 class="global-header__site-name">
+    <h1 class="global-header__site-name" aria-label="the accessible e store">
```

Comment:
Is it recommended to use aria-lable in conjunction with text inside an element? Wouldn't this result in the same/similar text being read twice?

Reply:
@aevanson, I looked at the bug that this fix is for and I couldn't reproduce it my machine. I understand your point, but I tested this out and VoiceOver only reads out what is in aria-label and ignores the child text element in the <a> tag.

Reply:
Thanks for looking into this @liskulic. I'll merge as-is!